### PR TITLE
Add search button to mobile top navbar

### PR DESCRIPTION
**Pull request by dfelczak**
Fixes: #839 

When search box is not active: 
screen shot 2017-11-17 at 00 15 34

and when search box is active: 
screen shot 2017-11-17 at 00 15 53

### CODE REVIEW

**Comment by Sekhmet** [comment]
Hey, it looks great!
I noticed that toggling between states takes a while, that's because it causes entire Wrapper to re-render:
image
Can you move this logic and state to Topnav itself, so it won't update entire Wrapper but only Topnav?

**Comment by dfelczak** [comment]
Nice catch! I will move state to Topnav :)

**Comment by dfelczak** [comment]
Hello @bonustrack, thanks for good feedback. I've just applied changes and PR is ready for next review.

File: src/components/Navigation/Topnav.js [hunk 1]
```diff
@@ -1,20 +1,58 @@
-import React from 'react';
+import React, { Component } from 'react';
```
Comment (Sekhmet) [inline src/components/Navigation/Topnav.js]: We don't import `Component` directly and use `React.Component` instead.

**Pull request merged**